terraform/router/nginx/main.tf

```diff
@@ -167,12 +167,13 @@ resource "kubernetes_deployment" "ingress-nginx" {
     template {
       metadata {
         labels = {
-          app     = "system"
-          name    = "ingress-nginx"
-          rack    = var.rack
-          system  = "convox"
-          service = "ingress-nginx"
-          type    = "service"
+          app            = "system"
+          name           = "ingress-nginx"
+          rack           = var.rack
+          system         = "convox"
+          service        = "ingress-nginx"
+          type           = "service"
+          proxy_protocol = var.proxy_protocol
```

Comment:
this new label will force the Deployment to restart to pick up the Nginx config-map update.